mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -75,6 +75,8 @@ def __init__(self, name: str, config: TunableDict):
                 raise ValueError("Must specify values for the categorical type")
             if self._range is not None:
                 raise ValueError("Range must be None for the categorical type")
+            if len(set(self._values)) != len(self._values):
+                raise ValueError("Values must be unique for the categorical type")
```

Comment:
Disallow repeats like `values: ["foo", "bar", "foo"]`

Reply:
Though I suppose we could offer support for `weighted` values at some other point ...

Reply:
yeah, let's think about the weighted values later - maybe in the context of the constraints